wiki/Ranking_Criteria/zh.md

```diff
@@ -44,69 +80,108 @@ _查看每个游戏模式具体的标准，请参阅 [osu!](/wiki/rc_osu!)，[os
 - **被渲染（Rendered）：** 通常指尚未在屏幕中完全消失的精灵图。
 - **osu! 像素（osu!pixel）：** 设计标签页中的最小尺寸。可以在编辑器屏幕的右上角看到，例如 `x: 104; y: 88`。
 
+
 ## 一般
 
 ### 规定
-
--   **两个物件不能出现在同一个时间点上（osu!mania 模式除外）。** 包括圈，滑条的开始和结束，转盘的开始和结束。虽然这么放物件仍然可以完成滑条或者转盘，但它与跟随歌曲节奏的目的相悖。
--   **不要滥用 Kiai 时间。** 一条经验法则是每分钟使用一次。Kiai 是为了强调歌曲高潮部分，所以不要经常使用它来获得绚丽的视觉效果。如果合适，你可以在歌曲高潮部分的中途将结束 Kiai，但每隔几个节拍 / 小节就使用 Kiai 是不合适的，
--   **不要手动编辑 `.osu` 文件中的编辑器无法修改的内容。** 例外是：`.osu` 自身的故事板，**滑条速度倍速（Slider velocity multipliers）** 和皮肤相关的选项，比如 SliderBorder 和 SliderTrackOverride。如果使用了非标准滑条速率倍速，它们必须在摸图时在谱面描述中指明。
+-   **两个物件不能出现在同一个时间点上。** 包括圈，滑条的开始和结束，转盘的开始和结束。osu!mania 模式除外。
 -   **背景 / 故事板 / 视频的内容不得有任何色情内容。** 包括裸体、近裸体、性暗示、暴力和滥用毒品等等。遵循 PG 级（普通级，建议在父母的陪伴下观看），适合 12 岁以上的人。图片的分级应该在这之上：能在所有观众的电视、公告标志上显示，且其内容不需要在任何国家进行审查。
--   **你必须为你的谱面的每个难度设置背景图片。** 如果谱面上有故事板，你可以轻松地淡出背景，但歌曲选择菜单、网站上的缩略图和选择不下载视频（如果谱面有视频）的玩家仍然需要谱面拥有背景图片。
--   **Approval 类别只适用于马拉松图。** 具有超过 5 分钟的 Draining 时间的长谱面符合这个类别。只有这种情况，这些谱面才能成为单难度谱面包。如果谱面的 Draining 时间小于 5 分钟，则需要完整的难度分布，且应该成为 Ranked 谱面。
--   **谱面文件夹中不能有任何未使用文件，** 除了谱面的 `.osb` 文件（因为即使谱面没有故事板，有时也会添加这些文件）和故事板的 `.thumb` 文件（因为它们是在图片文件夹中自动添加的）。未使用的文件会增加谱面的大小，这是没有必要的。
+-   **谱面文件夹中不能有未使用文件或 0 字节的文件。** 0 字节的文件会影响文件夹中其他文件的正常上传。这条规则的唯一例外是自动生成的 `.osb` 文件和故事版的 `.thumb` 文件。
+-   **不要通过编辑 `.osu ` 文件来修改难度设定或插入休息时间，这可能会对相应的游戏模式产生意外的问题。** 对 `.osu` 文件的其他修改是可以的，例如修改偏移系数（Stack leniency）,滑条速度（Slider velocity）,皮肤设定等等。
+-   **osu! 难度必须转换到其他游戏模式后不破坏星级 / PP。** 在某些情况下，`.osu` 文件可能格式不正确，导致转换后的难度无法正确显示星级并无法计算准确的 PP。
 
 ### 准则
-
--   **确保你可以通过谱面包中的每个难度。** 不断测试（玩）你的谱面是发现错误和更正问题的最佳方法之一。
--   **Kiai 应该在每一张谱面中保持一致。**（因为它们会在主菜单中播放。如果你的某个来自其他作图者的谱面使用了不同的 Kiai，只要适合歌曲，是可以的。
--   **Kiai 应该从一个白节奏点开始。**（或者更常见的是，从大的白节奏点 [下拍](/wiki/Glossary/#downbeat) 开始）。一般来说，歌曲的高潮部分从这里开始。
--   **歌曲不应该太长。** 最多 3 分钟，更长的话会感觉很累。如果你在剪短歌曲长度上需要帮助，请在谱面论坛的[一般问题](https://osu.ppy.sh/forum/56) 中提出。
--   **所有难度应该保持设置一致，例如 Letterboxing，倒计时和歌曲导入时间（AudioLeadIn）。** 在整个谱面包中使用相同的设置会看起来更专业，尽管不能绝对做到。一个谱面可能立即开始，在这时使用倒计时是不可能的，但使用歌曲导入时间是非常必要的；而另一个谱面可能在几秒种后开始，这时使用倒计时是可以的，但是使用歌曲导入时间迟是不合适的。
--   **osz 文件的大小最好在 10MB 以下，如果包含视频或故事板，最好在 30MB 以下。** 视频和故事板通常可以通过降低视频的质量和裁剪 / 调整故事板中元素的大小来减少尺寸。
+-   **所有谱面中的 Letterboxing 和倒计时设定应该一致。**
+-   **不能通过编辑 `.osu` 文件来修改滑条节拍倍率。** 大多数自定义值会导致不整齐的滑条节拍，虽然 0.5，1.333 和 1.5 这些值有一定的使用，但只能在比标准节拍速率更适合歌曲时才能使用。
+-   **Kiai 应该从歌曲中的一个声音开始。** 否则让 Kiai 听起来与歌曲无关。
```

Comment:
Kiai should start on a sound in the music. 
    Doing so otherwise causes the kiai flash to feel unrelated to the song.

否则让 Kiai 听起来与歌曲无关。=>
否则**会**让 Kiai 听起来与歌曲无关。

不加`会`字看上去改变了原文的意思